src/_locales/ur/messages.json

```diff
@@ -170,6 +182,14 @@
   },
 
   // context menu
+  "contextMenuItemConvertPageURL": {
+    "message": "اس صفحے کے لیے کیو آر کوڈ بنائیں",
+    "description": "Context menu item title for generating a QR code for the current page URL."
+  },
+  "contextMenuItemConvertPageURLAccessKey": {
+    "message": "&Generate QR code for this page",
```

Comment:
This is apparently missing a translation though hmm… cannot merge it as that, but I'll add a tag so it counts as accepted :wink:

Reply:
It's because i did not understand what the & symbol is doing.. Should i replace the entire text or leave the &Generate part (there are 5 of these that i missed.)

Reply:
I translated it all to Urdu (Maybe I should have left the word following the `&`?)

Reply:
As for the ampersand (`&?) symbol, what this means [is explained here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations). Basically it's for creating a keyboard hotkey.